### PR TITLE
Feature Flags

### PR DESCRIPTION
**Pull request by vjkoskela**
Added feature flags to control different aspects of the application: telemetry, hosts, expressions, alerts. Also added feature setting for telemetry to control proxy.

### CODE REVIEW

File: app/controllers/ApplicationController.java [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        _alertsEnabled = configuration.getBoolean("portal.features.alerts.enabled");
+    }
+
+    /**
+     * Render the header view.
+     *
+     * @return Rendered header view.
+     */
+    public F.Promise<Result> getHeaderViewModel() {
+        return F.Promise.pure(ok(views.html.HeaderViewModel.render(
```
Comment (BrandonArp) [inline app/controllers/ApplicationController.java]: Please make a features object that we can pass into the render function.  I don't like a function that takes half a dozen boolean parameters.  Any refactoring would be nearly impossible.

**Comment by BrandonArp** [comment]
Just the one comment and then I'm good.  It would be nice to have a completed "Features" object that is injectable. ( :

File: app/controllers/ApplicationController.java (hunk 1 above)
Comment (vjkoskela) [inline app/controllers/ApplicationController.java]: Although I agree whole heartedly with the sentiment I don't see how this makes refactoring impossible the code path was private and extremely well constrained to a single interaction (between controller and view). Nevertheless, the feature configuration has been modeled and is now injected.

**Comment by vjkoskela** [comment]
Updated review with injected Features instance.

**Comment by BrandonArp** [comment]
Perfect!  But you're missing the Features classes in the commit.

**Comment by vjkoskela** [comment]
Addresses #17.

**Pull request merged**